docs/notebooks/neural_dual.ipynb

```diff
@@ -12,7 +12,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def plot_ot_map(neural_dual, source, target, inverse=False):
</pre><p>For consistency, would rename <code>inverse=False -&gt; forward=True</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>